website/docs/installation.md

```diff
@@ -0,0 +1,5 @@
+---
+id: installation
+title: Installation
+sidebar_label: Installation
```

Comment:
No slug?

Reply:
Slug can be automatically generated from the file name - we could also explicitly add it to prevent breaking links when renaming files